src/components/Header/Header.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+
+export default function Header() {
+  return (
+    <div className='header'>
+      <ul>
+        <NavLink exact to={'/main'}>
+          <p>Home</p>
+        </NavLink>
+        <NavLink exact to={'/directors'}>
+          <p>Directors</p>
+        </NavLink>
+        <NavLink exact to={'/movies'}>
+          <p>Star Wars Movies</p>
+        </NavLink>
+        <NavLink exact to={'/viewers'}>
```

Comment:
for the paths you will only need them in a string, you can get rid of the {} around them. It looks like you have a /main here but not in your App.js